cmd/operator/main.go

```diff
@@ -54,6 +54,33 @@ const (
 	logFormatJson   = "json"
 )
 
+type namespaces map[string]struct{}
+
+// Set implements the flagset.Value interface.
+func (n namespaces) Set(value string) error {
+	ns := strings.Split(value, ",")
+	for i := range ns {
+		n[ns[i]] = struct{}{}
```

Comment:
nit: this will panic if `n` is uninitialized.

Reply:
What is the nit here? A map will also panic if you attempt to set a value when it is uninitialized. The solution is to `make(map...)` or `m := map...{}`. I think the same applies here, no?

Reply:
Sorry for being imprecise, the nit is that the following will panic:
```
var ns namespaces
ns.Set("foo")
```
Your suggestion is spot on, we may lazy initialize the map here in `Set` to make it safer.

Reply:
:+1: plz see the updated code